### PR TITLE
[OpenAI] Update release date

### PR DESCRIPTION
**Pull request by mssfang**
Prepare for the first beta release of OpenAI SDK for Java.

Clean up the sample structure and links

### CODE REVIEW

File: sdk/openai/azure-ai-openai/CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,5 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2023-05-22)
```
Comment (jpalvarezl) [inline sdk/openai/azure-ai-openai/CHANGELOG.md:3]: should we bump this to `beta.2`?

Comment (mssfang) [reply sdk/openai/azure-ai-openai/CHANGELOG.md:3]: Have we release the first beta?

Comment (jpalvarezl) [reply sdk/openai/azure-ai-openai/CHANGELOG.md:3]: The drop for LinkedIn, I thought. Or that doesn't count for this version count?

**Review by brandom-msft — APPROVED** [review]
lgtm - you may want to pull to get the changes we just merged, a bit of overlap but should automerge w/o issue.

File: sdk/openai/azure-ai-openai/CHANGELOG.md (hunk 1 above)
Comment (mssfang) [reply sdk/openai/azure-ai-openai/CHANGELOG.md:3]: Yeah. The one we shared with Linkedin is not released to maven but is our private EngSys repository internally. There is still no namespace existing in the maven repo.

**Review by jpalvarezl — APPROVED** [review]

**Comment by ZieglerHan** [comment]
when I try to use stream. I always got this error:  I beg your help.

```
Caused by: com.fasterxml.jackson.databind.exc.MismatchedInputException: Missing required creator property 'usage' (index 3)
 at [Source: (String)"{"id":"chatcmpl-7ORI3zwlnUA9cgGa4bU1V1I5XoJO0","object":"chat.completion.chunk","created":1686059131,"model":"gpt-35-turbo","choices":[{"delta":{"role":"assistant"},"index":0,"finish_reason":null}]}"; line: 1, column: 198] (through reference chain: com.azure.ai.openai.models.ChatCompletions["usage"])
	at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)
	at com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1755)
	at com.fasterxml.jackson.databind.deser.impl.PropertyValueBuffer._findMissing(PropertyValueBuffer.java:192)
```

https://github.com/Azure/azure-sdk-for-java/blob/bb37892275404e8a3cac3d4e2c639cbc9b18b9f9/sdk/openai/azure-ai-openai/src/samples/java/com/azure/ai/openai/usage/GetChatCompletionsStreamSample.java#LL53C35-L53C35

**Comment by mssfang** [comment]
> when I try to use stream. I always got this error: I beg your help.
> 
> ```
> Caused by: com.fasterxml.jackson.databind.exc.MismatchedInputException: Missing required creator property 'usage' (index 3)
>  at [Source: (String)"{"id":"chatcmpl-7ORI3zwlnUA9cgGa4bU1V1I5XoJO0","object":"chat.completion.chunk","created":1686059131,"model":"gpt-35-turbo","choices":[{"delta":{"role":"assistant"},"index":0,"finish_reason":null}]}"; line: 1, column: 198] (through reference chain: com.azure.ai.openai.models.ChatCompletions["usage"])
> 	at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)
> 	at com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1755)
> 	at com.fasterxml.jackson.databind.deser.impl.PropertyValueBuffer._findMissing(PropertyValueBuffer.java:192)
> ```
> 
> https://github.com/Azure/azure-sdk-for-java/blob/bb37892275404e8a3cac3d4e2c639cbc9b18b9f9/sdk/openai/azure-ai-openai/src/samples/java/com/azure/ai/openai/usage/GetChatCompletionsStreamSample.java#LL53C35-L53C35

@ZieglerHan Are you using OpenAI client or Azure OpenAI client?

Feel free to create an independent GitHub issue if necessary.

**Comment by mssfang** [comment]
Found issue created: https://github.com/Azure/azure-sdk-for-java/issues/35322

**Pull request merged**